src/pages/TwitterTimeline.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+
+class TwitterTimeline extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            posts: [],
+        };
+    }
+
+    componentDidMount() {
+        const script = document.createElement("script");
+
+        script.src = "https://platform.twitter.com/widgets.js";
+        script.async = true;
+
+        document.body.appendChild(script);
+    }
+
+    render(){
+        return (
+            <div>
+                <a
+                    class="twitter-timeline"
+                    href="https://twitter.com/ExpediaGroupEng?ref_src=twsrc%5Etfw"
```

Comment:
Just curious, what does this magic incantation`twsrc%5Etfw` do?

Reply:
I'm not too sure about that.

This is how the link is auto-generated from https://publish.twitter.com/.

It seems to be a mystery :p https://twittercommunity.com/t/no-api-documentation-for-ref-src/135946